Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -872,31 +873,43 @@ def get_entries_for_fetched_incident(ticket_id, should_get_comments, should_get_
 def fetch_incidents(query, id_offset, should_get_attachments, should_get_comments, should_mirror_in, should_mirror_out,
                     comment_tag, attachment_tag, fetch_by_created=None):
     last_run = demisto.getLastRun()
-    demisto.debug(f"last_run: {last_run}" if last_run else 'last_run is empty')
-    if last_run and last_run.get("idOffset"):
-        id_offset = last_run.get("idOffset")
+    demisto.debug(f'last_run: {last_run}' if last_run else 'last_run is empty')
+    last_created_time = ''
+    if last_run:
+        id_offset = last_run.get('idOffset') if last_run.get('idOffset') else ''
+        last_created_time = last_run.get('lastCreatedTime') if last_run.get('lastCreatedTime') else ''
     if not id_offset:
         id_offset = 0
 
     incidents, max_results = [], 50
     if id_offset:
         query = f'{query} AND id >= {id_offset}'
     if fetch_by_created:
-        query = f'{query} AND created>-1m'
+        if last_created_time:
+            last_issue_time = parse(last_created_time)
+            minute_to_fetch = last_issue_time - timedelta(minutes=1)
+            formatted_minute_to_fetch = minute_to_fetch.strftime('%Y-%m-%d %H:%M')
+            query = f'{query} AND created>={formatted_minute_to_fetch}'
```

Comment:
should the space in the value of `formatted_minute_to_fetch` should be escaped? or is that handled properly by the API?

Reply:
fixed it with \" \" and tested it in jira